lib/bundle/commands/cleanup.rb

```diff
@@ -19,7 +19,13 @@ def run
         taps = taps_to_untap
         if ARGV.force?
           if casks.any?
-            Kernel.system "brew", "cask", "uninstall", "--force", *casks
+            if ARGV.include?("--zap")
```

Comment:
`action = if ARGV.include?("--zap")`.

Does `zap` do an `uninstall` and `zap`?

Reply:
Yes, `zap` implies `uninstall` first.